### ISSUE TITLE
[Professions][Mining][BoreanTundra] Cobalt Nodes

### ISSUE THREAD

**Issue by TheRealestMcCoy** [dalaranwow/dalaran-wow#4638]
**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
This issue may be pending a update on the ptr being labeled under something else.
However cobalt seems to rare as it sits. I can fly around and basically only ever see the same node over and over 98% of the time.
**Expected behaviour**: Tell us what should happen instead
Cobalt should be more wide spread and avaliable.
**Steps to reproduce the problem**:

1. Fly around look for cobalt
2. Look for cobalt in places it should be but seem to never spawn.
3. 

**Include proofs for this behaviour**
This is the most current screen shot that i have for gatherer for cobalt in Borean Tundra. This is collected via me on 2 different chars. This also includes some people sharing their gather collects via addon. This also includes and guild members with the addon that collects them and are shared real time if I am on as they collect it.
![wowscrnshot_032817_121802](https://cloud.githubusercontent.com/assets/12252441/24415880/ec792b76-13b0-11e7-8b16-d6b797aab75d.jpg)

[167 Possible Nodes in Borean Tundra](http://www.wowhead.com/object=189978/cobalt-deposit)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
currently we have 91 possible spawn points for Cobalt Nodes (rich/normal) in the tundra. however those spawn are pooled, which means that there could either spawn a normal Cobalt or a Rich Cobalt.

moreover it's defined that there should be only 30 pools active at the same time. I will increase this to 91 and it should solve the issue

**Comment by Rushor** [comment]
you can see the pooling here https://searchcode.com/codesearch/view/3510548/

**Comment by Rushor** [comment]
DB/Gameobject: Borean Tundra Ore - Ore Cobalt + Rich Cobalt Pooling increased to 91 maxlimit

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4638
* commit: 4b4123285879794597c3cbc1b815b937502a8cd9

**Comment by Aridam** [comment]
Not sure where to comment in regards to there being a master-report for this or make one myself but, this still seems to be in-effect as Cobalt not just in the Borean Tundra but just-about across the board is scarce.

http://www.wow-professions.com/farming/cobalt-ore-farming -- following this guide I've painstakingly gone-through these routes only to be dumbfounded by how there's so little Cobalt, and it seems that Veins have a rather long respawn timer, likely longer than what it should be, because even after a full circle on a 150% Flying Mount, I come back only to see that no Veins whatsoever could be found again.

This seems to be a general issue as well because I've followed the same guidelines for Saronite and went to Icecrown, only to get the same result there as well, with no evident farmers around or seen en-route just-about anywhere to be farming, and even if otherwise, once-again the respawn timers seem too long as I made a circle there as well -- again, on my slow Mount -- only to get greeted by the nothingness before me the second time there.

Anyone to confirm/deny this?

**Comment by Ghostadder** [comment]
Is this problem solved or not because Cobalt still seems to be extremely rare and hard to find anywhere with the wotlk guides either.

**Comment by Ghostadder** [comment]
Even after 2 years it's still pending?

**Comment by Chenocide** [comment]
Yes, there's only 1 guy capable of merging to live, so you're gonna have to wait until he bothers fixing it.

**Comment by Ghostadder** [comment]
only 1 guy :/ well guess i have to wait.. I beg that he reads these comments..
